account/src/test/java/org/killbill/billing/account/dao/MockAccountDao.java

```diff
@@ -48,15 +51,12 @@
 import org.killbill.clock.Clock;
 import org.testng.Assert;
 
-import com.google.common.base.Predicate;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Iterables;
-import com.google.inject.Inject;
+import static org.killbill.billing.util.collect.CollectionTransformer.*;
```

Comment:
Nit: avoid wildcard imports.

Reply:
Ok, I'll address this together with next PR. I think I have several places that use willcard imports.

Reply:
Sounds good. FYI, if you are using IntelliJ, our default settings prevent these, check: https://github.com/killbill/killbill/blob/f182e0bdef9570044088191d9779bec7c8066081/.idea/codeStyleSettings.xml#L9-L10

Reply:
> Ok, I'll address this together with next PR. 

Err.. I changed my mind.. :sweat_smile: . I changed all willcard import in `util`, `tenant`, and `account`. Seems like they not as much as I thought it would be.

Reply:
> FYI, if you are using IntelliJ, ....

Ah I think I know. For some classes, I use feature "import static .... <class_name>.<method_name> ..." when Intellij showing up tips ballon.